src/orders/components/OrderPaymentOrTransaction/OrderPaymentOrTransaction.test.tsx

```diff
@@ -25,6 +25,7 @@ jest.mock("@saleor/macaw-ui", () => ({
   ResponsiveTable: jest.fn(() => <></>),
 }));
 jest.mock("@saleor/macaw-ui-next", () => ({
+  ...(jest.requireActual("@saleor/macaw-ui-next") as object),
```

Comment:
question: Is there any reason why you use actual implementation?

Reply:
Yes, it took me a while to realise that mock was the issue, but MacawUI components such as Box, Text etc. were unable to be rendered. Jest gives a very vague error and I would have to mock each and every macaw component that is used, it would be very time consuming.